content/library/api/connections/secrets-toml.md

```diff
@@ -0,0 +1,38 @@
+---
+title: secrets.toml
+slug: /develop/api-reference/connections/secrets.toml
+---
+
+## secrets.toml
+
+`secrets.toml` is an optional file you can define for your working directory or global development environment. When `secrets.toml` is defined both globally and in your working directory, Streamlit combines the secrets and gives precendence to the working-directory secrets. For more information, see [Secrets management](/develop/concepts/connections/secrets-management).
+
+### File location
+
+To define your secrets locally or per-project, add `.streamlit/secrets.toml` to your working directory. Your working directory is wherever you call `streamlit run`. If you haven't previously created the `.streamlit` directory, you will need to add it.
```

Comment:
Nit: "Working directory" isn't quite the right term, since you can change your working directory from inside Python. I think we should unify our docs behind a term like "base directory".

Reply:
This might be easier as a follow up so we can address it cleanly.